<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1852</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTT文章搜尋器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>關鍵字：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtKeyword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>請輸入關鍵字</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtContent">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>1271</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>這是第一行
這是第二行
哈啊哈啊</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>701</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜尋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSafeSearch">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>安全搜尋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取得原始資料</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncrypt1">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加密資料庫(1)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncrypt2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加密資料庫(2)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecrypt">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIndex">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>建立索引</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>690</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最大搜尋數目：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1852</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
